kiwi/system/prepare.py

```diff
@@ -99,10 +99,15 @@ def setup_repositories(self, clear_cache=False, signing_keys=None):
         repository_sections = \
             self.xml_state.get_repository_sections_used_for_build()
         package_manager = self.xml_state.get_package_manager()
+        locale_list = self.xml_state.get_locale()
         if self.xml_state.get_rpm_check_signatures():
             repository_options.append('check_signatures')
         if self.xml_state.get_rpm_excludedocs():
             repository_options.append('exclude_docs')
+        if locale_list:
+            repository_options.append(
+                '_install_langs%{0}'.format(':'.join(locale_list))
```

Comment:
This is wrong. It's supposed to be `%_install_langs <lang>:<lang>...`

Reply:
nope, not wrong. I just pass the information in that format to the Repository class. If you look further you find this:  https://github.com/SUSE/kiwi/pull/936/files#diff-55cc4623f69c01acca0ea339e5b6d70aR58

Reply:
so it gets resolved properly in the end, also the unit test shows that :) and I have tested it in real live runs a hundred times the last days :)

Reply:
Ah, OK.